app/controllers/users_controller.rb

```diff
@@ -1,7 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
   def show
-    @id = params[:id]
+    @user = User.find_by(id: params[:id])
```

Comment:
- [ ] the show actions should look something like `@user = User.find(params[:id])`

Reply:
@mikethreels Okay, I'll do it. but can you elaborate on what the difference is? since they're both working I mean